### PR TITLE
Test the login command

### PR DESCRIPTION
**Pull request by dmathieu**
Because only sanbashi tests is not enough

### CODE REVIEW

File: test/commands/login.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+describe('container login', () => {
+  beforeEach(() => {
+    cli.mockConsole()
+    process.env.HEROKU_API_KEY = 'heroku_token'
+    sandbox = sinon.sandbox.create()
+  })
+  afterEach(() => sandbox.restore())
+
+  it('logs to the docker registry', () => {
```
Comment (tt) [inline test/commands/login.js:19]: Should this also verify that the token is passed on standard in?

Comment (dmathieu) [reply test/commands/login.js:19]: You're right. Sinon is weird though. It doesn't fail when an argument is missing.

**Comment by dmathieu** [comment]
The coverage is "decreasing" because with this, we start counting all commands in the coverage.

File: commands/login.js [hunk 2]
```diff
@@ -24,9 +24,7 @@ async function login (context, heroku) {
   try {
     await dockerLogin(registry, password, context.flags.verbose)
   } catch (err) {
-    cli.error(`Error: docker login exited with ${err}`)
-    cli.hush(err.stack || err)
```
Comment (tt) [inline commands/login.js:28]: What did this do?

**Review by tt — APPROVED** [review]

File: commands/login.js (hunk 2 above)
Comment (dmathieu) [reply commands/login.js:28]: It appears it shows the javascript stack (for a docker error which seems weird).
The main reason I removed it is that `cli.exit()` exits the tests too, preventing us from properly testing.

`cli.error()` takes a second argument which is the exit code. But `cli.hush()` isn't documented.

**Pull request merged**